controllers/datadogagent/clusteragent.go

```diff
@@ -520,6 +520,31 @@ func newClusterAgentPodTemplate(logger logr.Logger, dda *datadoghqv1alpha1.Datad
 	return newPodTemplate, nil
 }
 
+// getClusterAgentAffinity returns the pod anti affinity of the cluster agent
+// the default anti affinity prefers scheduling the runners on different nodes if possible
+// for better checks stability in case of node failure.
+
```

Comment:
could you remove this line, 
comments need to be attached to the function to be used as documentation.

```suggestion
```